gui/quantum.lua

```diff
@@ -275,80 +225,57 @@ local function order_minecart(pos)
     quickfort_orders.enqueue_additional_order(quickfort_ctx, 'wooden minecart')
     quickfort_orders.create_orders(quickfort_ctx)
 end
+--Key Input
+function QuantumUI:onInput(keys)
```

Comment:
Unfortunately, this will break the ZScreen stack (that is, input will not be propagated properly). Try splitting out the Window widget into its own class and attaching input processing code there. Check out the example architecture in the docs here: https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#zscreen-class